### PR TITLE
Misc fixes to scripts

### PR DESCRIPTION
**Pull request by gmega**
This PR does a bunch of fixes to the Windows batch files:

1. using the correct convention for newline (^M);
2. removing `utils.bat` as [symbols defined in a batch file don't get exported](https://stackoverflow.com/questions/19798777/accessing-batch-functions-in-another-batch-file) so this was not doing much really;
3. adds a workaround so curl calls are more robust in the presence of third-party antiviruses.

It also does two changes in the Linux/Windows scripts:
1. add permissive CORS so the UI (app.codex.storage) can be used;
2. sets a bigger default TTL for block garbage collection (as per @veaceslavdoina request).

Yes, I know, it's a lot of intermingled stuff in a single PR, feel free to call me names. 🙂

### CODE REVIEW

File: scripts/download.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ else
     ARCHIVE_EXT=".tar.gz"
     EXE_EXT=""
 fi
-VERSION="v0.1.4"
+VERSION="v0.1.5"
```
Comment (emizzle) [inline scripts/download.sh:51]:
```suggestion
VERSION="v0.1.6"
```

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 services:
   # Codex Node
   codex:
-    image: codexstorage/nim-codex:0.1.4
+    image: codexstorage/nim-codex:0.1.5
```
Comment (emizzle) [inline docker-compose.yaml:4]:
```suggestion
    image: codexstorage/nim-codex:0.1.6
```

File: scripts/download_online.sh [hunk 3]
```diff
@@ -48,7 +48,7 @@ else
     ARCHIVE_EXT=".tar.gz"
     EXE_EXT=""
 fi
-VERSION="v0.1.4"
+VERSION="v0.1.5"
```
Comment (emizzle) [inline scripts/download_online.sh:51]:
```suggestion
VERSION="v0.1.6"
```

File: scripts/run_client.sh [hunk 4]
```diff
@@ -36,7 +36,7 @@ if [ ! -f eth.key ]; then
 fi
 
 # Set variables
-VERSION="v0.1.4"
+VERSION="v0.1.5"
```
Comment (emizzle) [inline scripts/run_client.sh:39]:
```suggestion
VERSION="v0.1.6"
```

**Review by emizzle — APPROVED** [review]
🙏

**Comment by veaceslavdoina** [comment]
Updated Codex version to the `v0.1.6` release and updated marketplace entries I've found.

**Pull request merged**